src/app/lib/config/toggles/localConfig.js

```diff
@@ -3,6 +3,9 @@ export default {
   ads: {
     enabled: false,
   },
+  ampAds: {
+    enabled: false,
```

Comment:
I think AMP Ads are worth keeping toggled on for LOCAL and maybe TEST environments so when people are developing amp-ads they don't have to update this value.

Reply:
Yep, agreed with this - let's have them enabled on local and test, then this PR looks good to me!

Reply:
Thanks. This has been done